pkgs/applications/networking/sync/bsync/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
```

I don't think the `rec` is necessary here

```suggestion
stdenv.mkDerivation {
```